### DISCUSSION TITLE
IBAX duyurusu hakkında

### DISCUSSION THREAD

**Discussion by analist45** [IBAX-io/go-ibax#953] [Soru Cevap - Türkçe] [answered]
IBAX tweet duyurusunda cüzdanlarınızı yedekleyin gibisinden bir paylaşım yaptı acaba 1 şubatta web arayüzü değişecek mi ?

**Comment by Yusufknk** [comment, marked as answer]
Hayır, cüzdana farklı cihaz ve tarayıcılardan tekrar erişebilmek için private keylerinizi alın diye bir uyarı
